### ISSUE TITLE
python setup.py clean should clean auxiliary files created by running tests, e.g. .pyc files

### ISSUE THREAD

**Issue by riastradh-probcomp** [probcomp/bayeslite#93]

**Comment by riastradh-probcomp** [comment]
It's not the same as the Debian packaging issue but it's similar.

The Debian packaging issue is that `python setup.py clean` does not clean everything that `python setup.py build` built -- in particular, `src/grammar.py`, `src/grammar.sha256`, and `src/version.py`.

The test issue is that `python setup.py clean` does not clean everything that `./check.sh` or `python setup.py test` created -- in particular, `tests/*.pyc`.

**Comment by gregory-marton** [comment]
Is it hard for clean to clean all those, for some reason?

**Comment by riastradh-probcomp** [comment]
No, it should be pretty easy, although it would be nice, as a perhaps separate but related issue, if running the tests in a read-only tree worked -- that would obviate the need for `python setup.py clean` to delete tests/*.pyc.
